dev/bots/custom_rules/protect_public_state_subtypes.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(nate-thegrate): remove this file if @protected changes, or add a test if it doesn't.
+// https://github.com/dart-lang/sdk/issues/57094
```

Comment:
I think the conversation there is pretty interesting:

- https://github.com/dart-lang/sdk/issues/57094